packages/rest/src/lib/RequestManager.ts

```diff
@@ -293,7 +293,11 @@ export class RequestManager extends EventEmitter {
 		const { url, fetchOptions } = this.resolveRequest(request);
 
 		// Queue the request
-		return handler.queueRequest(routeId, url, fetchOptions, { body: request.body, files: request.files });
+		return handler.queueRequest(routeId, url, fetchOptions, {
+			body: request.body,
+			files: request.files,
+			auth: request.auth !== false,
```

Comment:
Couldn't this be `!!request.auth`?

Reply:
If anything, it would be `!request.auth`. I guess there's a reason why vlad didn't make it that from the start

Reply:
Nop, and here's why
![image](https://user-images.githubusercontent.com/17960496/155213509-320c5fcc-a67a-4b97-807d-4bbb1c695d87.png)
auth should be set to `false` only if it was provided as `false`